src/components/EditableHeading/EditableHeading.jsx

```diff
@@ -246,7 +261,10 @@ EditableHeading.defaultProps = {
   ignoreBlurClass: undefined,
   onFinishEditing: undefined,
   onCancelEditing: undefined,
-  onIgnoreBlurEvent: undefined
+  onIgnoreBlurEvent: undefined,
+  style: undefined,
+  dataTestId: "",
```

Comment:
add a default please

Reply:
@mayaAssayag 
According to react it should either be undefined or CSSObject type